### DISCUSSION TITLE
Supabase start failing to apply migration

### DISCUSSION THREAD

**Discussion by spoken-angus** [supabase/supabase#11815] [Questions] [answered]
I'm following the module on [managing environments](https://supabase.com/docs/guides/resources/supabase-cli/managing-environments) and generated an initial migration. However when I try to run `supabase start` I'm getting this error.
```
Applying migration 20230120200536_remote_commit.sql...
Error: ERROR: schema net is not a member of extension "pg_net" (SQLSTATE 55000)
Try rerunning the command with --debug to troubleshoot the error.
```
I've even tried commenting out the pg_net stuff and then get this error. Any help would be amazing!
```
Applying migration 20230120200536_remote_commit.sql...
Error: ERROR: function graphql.get_schema_version() does not exist (SQLSTATE 42883)
Try rerunning the command with --debug to troubleshoot the error.
```

**Comment by GaryAustin1** [comment, marked as answer]
https://github.com/supabase/supabase/issues/10986
I don't think this is directly your issue, but it might give you a clue as you mentioned the schemas involved.
